content/applications/general/payment_acquirers/adyen.rst

```diff
@@ -0,0 +1,108 @@
+=====
+Adyen
+=====
+`Adyen <https://www.adyen.com/>`__ is a Dutch-based company that allows businesses to
+accept **online** and **point-of-sale** payments.
+
+.. image:: media/adyen_logo.png
+   :align: center
+   :alt: Adyen logo
+
+This Payment Acquirer offers additional options that are not available for other :doc:`Payment
+Acquirers <../payment_acquirers>`, such as the ability to refund partially a payment directly from
+Odoo, to capture the payment after the delivery, or to link a point of sale's terminal.
+
+Payment Acquirer Configuration
+==============================
+
+If not done yet, `contact Adyen <https://www.adyen.com/contact/sales>`__ to get your live account.
```

Comment:
To proceed with... you need an Adyen account in the live environment (rephrase this).